_data/D3nii/D3nii.md

```diff
@@ -0,0 +1,7 @@
+---
+name: Danyal Jamil
+institution: FAST NUCES Lahore
+profile_pic: Menoice.jpeg

```

Comment:
Incorrect file name, please do one of the following:
1. Change `Menoice.jpeg` to `D3nii.jpg ` in the markdown file.
2. Rename `D3nii.jpg ` to `Menoice.jpeg`.
I'd suggest you make the change in the markdown file!

Reply:
I changed it.